security/s2a/testdata/README.md

```diff
@@ -1,19 +1,19 @@
 ## Key Generation
-The private keys were generated using
+### CA Key/Cert Generation
 ```
-openssl genrsa -out <FILE_NAME> 4096
+openssl genrsa -out ca.key 4096
+openssl req -new -x509 -key ca.key -sha256 -subj "/C=US/ST=NJ/O=CA, Inc." -days 365 -out ca.cert
 ```
-The same private key was used for the root certificate and client certificate.
-
-The client/root certificates were generated using
+### Client Key/Cert Generation
 ```
-openssl req -new -x509 -key <KEY_FILE_NAME> -sha256 -subj "/C=US/ST=NJ/O=CA, Inc." -days 365 -out <CERT_FILE_NAME>
+cp ca.key client.key
+openssl req -new -x509 -key client.key -sha256 -subj "/C=US/ST=NJ/O=CA, Inc." -days 365 -out client.pem
 ```
+Note that the same private key was used for the client and ca.
```

Comment:
Any specific reason why we're doing this? If the same key is used, then the client is basically the CA :)

Can we chain the client cert from the CA cert the same we do for the server? We would then have similar commands for generating client and server certs. The difference would be the filenames.

Reply:
No specific reason, I just couldn't figure out how to chain the client cert from the CA cert (I'm not very familiar with openssl). I didn't realize I could just do the same as what I did for the server.

I updated the PR with the new instructions and client key/pem pair.